<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>8 Puzzle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="counter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ESTADO SOLUCIÓN</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="Ficha1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Ficha6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Ficha5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="emptyButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: transparent; /* Fondo transparente */
    border: none; /* Sin borde */
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Ficha7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Ficha8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="Ficha2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="Ficha3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Ficha4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>168</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
 {
	/* Color base del botón (lila) */
	background-color: #a8a8f0; 
	/* Color de texto en estado normal (negro) */    
	color: #000000; 
  	 /* Ajusta el radio del borde */
	border-radius: 5px; 
    padding: 5px;
}

QPushButton:hover:!pressed 
{
   	/* Color cuando pasas el cursor (morado) */
	background-color: #6a6ad8; 
	/* Color de texto en hover (blanco) */    
	color: #ffffff; 
}

QPushButton:pressed
{
	/* Color cuando presionas el botón (morado) */
	background-color: #6a6ad8; 
}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bell MT</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover {
    border: 2px solid #5a5a5a; /* Borde al pasar el cursor */
    border-radius: 4px; /* Ajuste para bordes redondeados */
}</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
